Automattic/wp-calypso
# update Newspack Blocks to v1.60.0

### Proposed Changes

* Updates Newspack Blocks in Editing Toolkit from v1.37.1 to v1.60.0. This should resolve #71842 as well as bring the fix from https://github.com/Automattic/newspack-blocks/pull/1173 to ETK.

With props to @noahtallen for helping us debug and update the sync script now that Newspack Blocks uses TypeScript in its source.

### Calypso updates to get it working
- Update peer dependencies, and remove a bunch of old ones we don't need any more.
- Remove swiper patch since it has no effect any more. Also remove prettier resolution which does nothing.
- Pin the stripe-js version to avoid updating it and breaking types.
- Add missing swiper dependency to a handful of packages.
- Force resolution of `wordpress` deps for newspack blocks. Otherwise, they'll get out of sync with the repo and pull in updates which break Calypso.
- Sync index.d.ts file from newspack-blocks, and adjust it to work with Calypso.
- Pin new swiper version in ETK to match newspack-blocks. Otherwise, the ETK build will resolve the repo-wide version, which is 4.5. This is because we don't consume newspack blocks as an npm dependency, so yarn/webpack doesn't resolve its dependencies normally. Instead, when linking newspack blocks code, the dependency version from ETK is used if it exists.

### Testing Instructions



* Confirm that the Blog Posts block continues to work, and that #71842 and https://github.com/Automattic/newspack-blocks/pull/1173 are resolved.
* Verify that wpcom phpunit tests pass before merging

#### Pre-merge Checklist





Related to #
